api/proto/metrics/metrics.proto

```diff
@@ -468,6 +468,42 @@ message ZMetricVolume {
   uint64 freeBytes = 9;
 }
 
+// Metrics for a snapshot
+// When a snapshot is created, its disk space is initially shared between
+// the snapshot and the file system, and possibly with previous snapshots.
+// As the file system changes, disk space that was previously shared becomes
+// unique to the snapshot, and thus is counted in the snapshot's used property.
+// Additionally, deleting snapshots can increase the amount of disk space
+// unique to (and thus used by) other snapshots.
+message ZMetricSnapshot {
+  // Snapshot UUID
+  string uuid = 1;
```

Comment:
Makes sense calling this field snapshot_uuid as well.

Reply:
Corrected